package.json

```
@@ -26,9 +26,11 @@
     "jest:update": "jest --updateSnapshot",
     "jest:watch": "jest --watch",
     "flow": "flow",
```

```suggestion
     "flow": "npm run flow-check-ios && npm run flow-check-android",
```
And then we can keep the `test` line the same as it already was. Doing this means we still have a single command to run *all* Flow things, which is handy.

---

I think we shouldn't do this. `npm run flow` is useful as is because:
- It uses and existing flow server so that subsequent runs are lightning quick (albeit targeting only ios)
- It allows us to do stuff like `npm run flow -- stop` / `npm run flow -- start` (and any other arguments `flow` supports)
- It aligns with the react-native codebase